browser/brave_ads/tooltips/ads_tooltips_controller.cc

```diff
@@ -9,7 +9,7 @@
 #include <utility>
 
 #include "brave/browser/ui/brave_tooltips/brave_tooltip_attributes.h"
-#include "brave/browser/ui/brave_tooltips/brave_tooltip_popup.h"
+#include "brave/browser/ui/views/brave_tooltips/brave_tooltip_popup.h"
```

Comment:
As I understand we shouldn't include header from views here.
@simonhong @emerick WDYT?

Reply:
Oops, I missed this. Right. This also should be removed from this file.

Reply:
Thanks for pointing this out, fixed.